### ISSUE TITLE
On the user accounts page, the delete button is not mentioned as the delete button 

### ISSUE THREAD

**Issue by poojaustad** [UstadMobile/UstadMobile#345]
**Describe the bug**
On the user accounts page, the delete button is not mentioned as the delete button . 

**To Reproduce**
1. Enable talk back on the device
2. Multiple accounts are accessed in the ustad app
3. On the user account page the delete button icon is mentioned as - A button 

**Expected behavior**
Delete icon should be mentioned as the delete button 

![2022-06-27_11h47_16](https://user-images.githubusercontent.com/78338755/175889664-d7e8a1b6-c9da-40a3-bd79-71a1a8a98aba.png)

Noted on android device
